csharp/TheAccountClass/TheAccountClass/Account.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TheAccountClass
+{
+    public class Account
+    {
+        private double _balance;
+        private int _accountNumber;
+        private Customer _owner;
+        private static int _accountQty = 0;
+
+        public Account(Customer owner) {
+            _balance = 0;
+            _accountNumber = ++_accountQty;
+            _owner = owner;
+        }
+
+        public void Credit(double amount)
+        {
+            _balance += amount;
+        }
+        public void Credit(double amount, Account account)
+        {
+            _balance += amount;
+            account._balance -= amount;
+        }
+        public void Debit(double amount)
+        {
+            _balance -= amount;
+        }
+        public void Debit(double amount, Account account)
```

Comment:
same as above:
this method can be wrapped inside a Service Class, like PaymentService
SubmitPayment(Account from, Account to, double amount);